shared/src/main/scala/fr/hmil/roshttp/HttpRequest.scala

```diff
@@ -24,6 +24,7 @@ final class HttpRequest  private (
     val port: Option[Int],
     val protocol: Protocol,
     val queryString: Option[String],
+    val credentials: Boolean,
```

Comment:
This name doesn't describe the effect very well. Change that to something like `crossDomainCookies`.

Reply:
DONE.